content/software/ide-v2/tutorials/01.getting-started-ide-v2/ide-v2-autocomplete-feature.md

```diff
@@ -0,0 +1,132 @@
+
+### Sketchbook
+
+![Arduino Sketchbook.](assets/local-sketchbook.png)
+
+Your sketchbook is where your code files are stored. Arduino sketches are saved as `.ino` files, and must be stored in a folder of the exact name. For example, a sketch named `my_sketch.ino` must be stored in a folder named `my_sketch`.
+
+Typically, your sketches are saved in a folder named `Arduino` in your `Documents` folder. 
+
+To access your sketchbook, click on the **file icon** located in the left hand menu. 
+
+### Board Manager
+
+![Board Manager.](assets/board-manager.png)
+
+With the board manager, you can browse and install packages, or "cores" for your boards. A board package is always required when compiling and uploading code for your board.
+
+There are several Arduino board packages available, such as **avr, samd, megaavr, samd** and more.
+
+***To learn more about the board manager, visit the [Installing new boards tutorial](/software/ide-v2/tutorials/ide-v2-board-manager).***
+
+### Library Manager
+
+![Library Manager.](assets/library-manager.png)
+
+With the library manager you can browse and install thousands of libraries. Libraries are extensions of the Arduino API, and makes it easier to for example control a servo motor, read specific sensors, or use a Wi-Fi module.
+
+***To learn more about the library manager, visit the [Installing libraries tutorial](/software/ide-v2/tutorials/ide-v2-installing-a-library).***
+
+### Serial Monitor
+
+![Serial Monitor.](assets/serial-monitor.png)
+
+The Serial Monitor is a tool that allows you to view data streaming from your board, via for example the `Serial.print()` command. 
+
+Historically, this tool has been located in a separate window, but is now integrated with the editor. This makes it easy to have multiple instances running at the same time on your computer.
+
+***To learn more about the Serial Monitor, visit the [Serial Monitor tutorial](/software/ide-v2/tutorials/ide-v2-serial-monitor).***
+
+### Serial Plotter
+
+![Serial Plotter.](assets/potentiometer-plotter.gif)
+
+The Serial Plotter tool is great for visualizing data using graphs, and to monitor for example peaks in voltage. 
+
+You can monitor several variables simultaneously, with options to enable only certain types. 
+
+***To learn more about the Serial Plotter, visit the [Serial Plotter tutorial](/software/ide-v2/tutorials/ide-v2-serial-plotter).***
+
+### Debugging
+
+![Debugger tool.](assets/playpause.gif)
+
+The debugger tool is used to test and **debug** programs, hence the name. It can be used to navigate through a program's execution in a controlled manner. 
+
+***To learn more about the debugger tool, visit the [Debugging tutorial](/software/ide-v2/tutorials/ide-v2-debugger).***
+
+### Autocompletion
+
+![Autocompletion tool.](assets/autocomplete.png)

```

Comment:
This screenshot does not show the autocomplete feature. Instead, it shows the "parameter hints" feature, which does not provide any sort of completion capabilities.

This is what the autocomplete feature looks like:
![image](https://user-images.githubusercontent.com/8572152/177313447-98f6ea53-5785-4a4d-88a9-4270d1f53001.png)